docs/cloud/provision/cloud-instance-guidance/README.md

```diff
@@ -19,6 +19,26 @@ We have profiled using single event transactions for the production rated cluste
 
 The **Working Set** is the number of streams being read from and written to concurrently. It's essential to recognise that writing one million events into one million streams is a very different scenario than writing one million events into a single stream.
 
+### Performance vs self hosted
+
+When load testing an application against Event Store Cloud EventStoreDB clusters, performance may differ from a self hosted solution when utilizing a similar instance size.
+
+Event Store Cloud utilizes ZFS to ensure filesystem integrity/safety as well as block level compression, to ensure the minimization of IOPs and a lower storage cost for data hosted.
+
+ZFS requires additional CPU and memory to provide these capabilities, as such a self hosted comparison should utilize a similar configuration, in order to provide a relevant comparison.
+
+### Micro (F1) instance sizes
+
+This rating is meant to enable a low cost point of entry for developers experimenting with Event Store Cloud, and to satisfy the use case of some customers that require multi-tenancy and whose application presents an extremely low volume workload (10-100) events a day per database (cluster). This size utilizes a burstable CPU class, which is how we achieve this goal across all supported public clouds.
+
+Due to the burstable CPU class, CPU shares are limited, this results in the following implications:
+
+- EventStoreDB runs on the .Net runtime (managed) this means CPU time is required for garbage collection operations, if the shares are not available and allocations are increasing this may result in client timeouts to the cluster, or an OOM condition.
+- CPU shares are required to maintain the cluster state topology via constant gossip message propagation. CPU shares are utilized to maintain cluster state even when the cluster is not under load.
+- EventStoreDB requires CPU cycles to maintain indexes, and for other maintenance processes, this means that performance may appear to degrade over time with growth of the database.
```

Comment:
EventStoreDB requires CPU cycles to maintain indexes, and for other maintenance processes. Our guidelines are based on fully populated systems with consistent throughput and the requisite CPU workload required to maintain them. Underpopulated or underloaded or systems (such as demo or development setups) will overperform compared to these guidelines. This sometimes creates incorrect expectations that do not match the results of performance/load tests on more fully populated systems.

Reply:
I believe I've updated the wording to reflect this.